src/Formatter/Formatter.php

```diff
@@ -15,6 +15,7 @@
 use Vanare\BehatCucumberJsonFormatter\Printer\FileOutputPrinter;
 use Vanare\BehatCucumberJsonFormatter\Renderer\JsonRenderer;
 use Vanare\BehatCucumberJsonFormatter\Renderer\RendererInterface;
+use Bex\Behat\ScreenshotExtension\Event\ScreenshotUploadCompleteEvent;
```

Comment:
@cawolf I am not a Symphony developer, so I need a bit of advise here. Is there a way to have a soft dependency on this package to avoid errors for people that do not use it?